Formula/httplab.rb

```diff
@@ -0,0 +1,21 @@
+class Httplab < Formula
+  desc "Interactive Web Server"
+  homepage "https://github.com/gchaincl/httplab"
+  url "https://github.com/gchaincl/httplab/archive/v0.2.1.tar.gz"
+  sha256 "6a4a7b8fecd34cb95a8281d0f1f4fafdd78b23121e34ec64c7d294559e9ff207"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/gchaincl/httplab").install buildpath.children
+    cd "src/github.com/gchaincl/httplab" do
+      system "go", "build", "-o", bin/"httplab"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/httplab", "-version"
```

Comment:
Can’t we write a test that does something more substantial?

Reply:
I'd like to see this merged; I'm happy to help if I can, but I'm inclined to agree that I find it hard to imagine what a good test would look like here: httplab is interactive by design.

Reply:
there are some unit test, is this about running them?

Reply:
Something like this should do it:
```ruby
 (testpath/"test.exp").write <<~EOS
        spawn #{bin}/httplab
        sleep 1
        send \003
        expect eof
      EOS
      system "expect", "-f", "text.exp"
end
```